backend/layers/common/entities.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import List
+
+
+@dataclass
+class DatasetStatus:
+    status: str  # TODO: use an enum
+
+
+@dataclass
+class DatasetArtifact:
+    type: str
+    uri: str
+
+
+@dataclass
+class DatasetMetadata:
+    organism: str
+    tissue: str
+    assay: str
+    disease: str
+    sex: str
+    self_reported_ethnicity: str
+    development_stage: str
+    cell_type: str
+    cell_count: int
+
+
+@dataclass
+class Dataset:
+    id: str
+    status: DatasetStatus
+    metadata: DatasetMetadata
+    artifacts: List[DatasetArtifact]
+
+
+@dataclass
+class CollectionMetadata:
+    name: str
+    description: str
+    owner: str
+    contact_name: str
+    contact_email: str
+    links: List[dict]  # TODO: use a dataclass
+
+
+@dataclass
+class Collection:
```

Comment:
Did we decide that we could add a `latest_version_id` to Collection?  Or are we okay finding latest version via max/sort/iteration?

Reply:
See my comment above - this class isn't tied to the `Collection` table. I can see the confusion though so we should probably rename this to something better. Unfortunately, the only name that comes to mind right now is `CollectionData`...